src/ToolbarService.php

```diff
@@ -102,7 +102,7 @@ public function registry(): PanelRegistry
      */
     public function isEnabled(): bool
     {
-        if (isset($GLOBALS['__PHPUNIT_BOOTSTRAP'])) {
+        if (defined('PHPUNIT_COMPOSER_INSTALL') && !$GLOBALS['FORCE_DEBUGKIT_TOOLBAR']) {
```

Comment:
Of course they break my hacks :laughing:

Reply:
Just re-introduced a new hack 😁